### DISCUSSION TITLE
How to set reasonable 'irrad_frac_refl_bol' and 'gravb_bol' ?

### DISCUSSION THREAD

**Discussion by Yzxastr** [phoebe-project/phoebe2#692] [Questions]
Hello! I've set the primary temperature of the T1 = 6700K, irrad_frac_refl_bol=0.5, gravb_bol=0.32, but it didn't pass the check, and PHOEBE was suggested that binarys with temperatures above 6600K should be set in the 0-1 range for the 'irrad_frac_refl_bol'. I would like to ask which literature was used to set this range? Whether to treat this binary as a binary with radiative envelope? In general, we use 7200K as the boundary between the convective envelope and the radiative envelope.

**Comment by aprsa** [comment]
Setting those three parameters should not have tripped the checks; you might be seeing a warning -- phoebe tries to give you a "heads up" when you are in the range of temperatures where defaults might not be appropriate. You are of course free to ignore it and phoebe will still compute it for the values of parameters that you set. I tried quickly to reproduce the check failure using your values but couldn't, so in order to better help you, could you please post your entire script that's tripping the checks?
